migrate_pim/upgrade_major_version.rst

```diff
@@ -0,0 +1,32 @@
+How to upgrade to a major version?
+==================================
+
+We release a major version each year, these new major versions bring new features and larger changes to answer clients growing needs.
+
+Migrate from 1.7 to 2.0
+------------------------------
+
+To migrate from 1.7 to 2.0, we recommend the use of our brand new migration tool `Transporteo`_.
+
+.. _Transporteo: https://github.com/akeneo/transporteo
+
+We're continuously improving Transporteo to cover more and more use cases and automate more and more the migrations.
+
+Migrate from 2.3 to 3.0
+------------------------------
+
+To migrate from 2.0 to 3.0 version, you can simple follow the migration guides:
```

Comment:
We should explain that the user has to migrate on 2.1, then 2.2, then 2.3 to migrate on 3.0:
To migrate from 2.0 to 3.0 version, you first need to go through minor versions. For instance, if you are on the 2.0 version, you will have to migrate to the 2.1, then the 2.2 and finally the 2.3. From the 2.3 version, you will be able to follow this migration guide:

Reply:
Totally agree ! Thanks for that, I change it. :+1: